src/test/java/utilities/DriverConfiguration.java

```diff
@@ -7,37 +7,69 @@
 import java.net.URL;
 import java.nio.file.Paths;
 import java.util.Objects;
+
+import org.openqa.selenium.Dimension;
 import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.remote.DesiredCapabilities;
 
 public class DriverConfiguration {
+
   public WebDriver getDriver() {
-    if (Objects.equals(LocalEnviroment.getPlatform(), "Web")) {
-      ChromeDriver driver = new ChromeDriver();
-      driver.manage().window().maximize();
+    if (LocalEnviroment.getPlatform().equalsIgnoreCase("Web")) {
+      WebDriver driver = configureWebDriver();
+      Dimension windowResolution = ScreenResolution.getResolutionFromEnv();
+      if (Objects.nonNull(windowResolution))
+        driver.manage().window().setSize(windowResolution);
       driver.get(LocalEnviroment.getUrl());
       return driver;
-    } else if (Objects.equals(LocalEnviroment.getPlatform(), "Android")) {
+    } else if (LocalEnviroment.getPlatform().equalsIgnoreCase("Android")) {
       try {
         URL url = new URL("http://127.0.0.1:4723");
         return new AndroidDriver(url, fillCapabilities());
 
       } catch (MalformedURLException e) {
         throw new RuntimeException(e);
       }
-    } else return null;
+    } else {
+      return null;
+    }
+  }
+
+  private WebDriver configureWebDriver() {
+    WebDriver driver;
+    String browser = LocalEnviroment.getBrowser();
+    if (browser == null) {
+      browser = "chrome";
+    } else {
+      browser = browser.toLowerCase();
```

Comment:
remove unnecesary else. Put directly when browser variable is declared as String browser = LocalEnviroment.getBrowser().toLowerCase